database/sqlite3/sqlite3_test.go

```diff
@@ -181,3 +182,35 @@ func TestMigrateWithDirectoryNameContainsWhitespaces(t *testing.T) {
 	}
 	dt.Test(t, d, []byte("CREATE TABLE t (Qty int, Name string);"))
 }
+
+var pathTests = []struct {
+	in  string
+	out string
+}{
+	// simple valid path, no whitespaces
```

Comment:
Can you also add a test for path with a preceding `/`?

Reply:
Do you mean just a preceding `/` without the schema?  As in `/This/For A/Path.db` ?

Or do you mean a malformed url string like `/sqlite3:///This/For A/Path.db` ?

Reply:
Sorry, I meant a path **without** a preceding `/`, which is not what I typed... 🤦 
So a string like: `sqlite3://This/For A/Path.db` => `This/For A/Path.db`